src/TechObject/TechObject.cs

```diff
@@ -185,7 +185,7 @@ public string SaveRestrictionAsLua(string prefix)
         {
             string res = "";
             string tmp = "";
-            string comment = "\t\t--Объект №" + getN(this);
+            string comment = $"\t\t--Объект №{getN(this)} ({this.Name})";
```

Comment:
Предлагаю упростить:
```suggestion
            string comment = $"\t\t-- ({this.Name})({this.Number})";
```

Reply:
Скорректировано